modules/util/util.go

```diff
@@ -256,3 +256,8 @@ func ToFloat64(number any) (float64, error) {
 	}
 	return value, nil
 }
+
+// ToRef returns the reference of a copy of any given value
+func ToRef[T any](val T) *T {
```

Comment:
A nit, I think Golang's concept for such is "pointer" but not "reference" ?

Reply:
it's technically the same ... but I go with the go-slang

in the golang source they also often refer to it as reference ...

but the type is https://pkg.go.dev/unsafe#Pointer

Reply:
`reference` is usually used to describe the behavior, the concept itself I meant is like this https://go.dev/tour/moretypes/1

> Pointers
> Go has pointers. A pointer holds the memory address of a value. 


And I meant maybe the `ToRef` could be renamed to `ToPointer` (not only the comment), anyway, it's only a nit and doesn't block.